app/scripts/modules/core/src/pipeline/manualExecution/layout/ManualExecutionFieldLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { IValidationMessageProps, ValidationMessage } from 'core/validation';
+import { IFieldLayoutProps } from 'core/presentation';
+import { isUndefined } from 'util';
+
+export class ManualExecutionFieldLayout extends React.Component<IFieldLayoutProps> {
+  public render() {
+    const { label, help, input, actions, touched, validationMessage, validationStatus } = this.props;
+
+    const showLabel = !isUndefined(label) || !isUndefined(help);
```

Comment:
Not sure if there's any chance of these being `null` or empty strings, if there is maybe `_.isEmpty` would be safer?

Reply:
👍